samples/KafkaFlow.Sample/ConsoleLogHandler.cs

```diff
@@ -8,19 +8,21 @@ public class ConsoleLogHandler : ILogHandler
         public void Error(string message, Exception ex, object data)
         {
             Console.ForegroundColor = ConsoleColor.Red;
-            Console.WriteLine($"Kafka Error: {message} | Exception: {JsonSerializer.Serialize(ex)}");
+            Console.WriteLine($"\nKafka Error: {message} | Exception: {JsonSerializer.Serialize(ex)}");
             Console.ForegroundColor = ConsoleColor.White;
         }
 
         public void Info(string message, object data)
         {
-            Console.WriteLine($"Kafka Info: {message} | Data: {JsonSerializer.Serialize(data)}");
+            Console.ForegroundColor = ConsoleColor.Yellow;
```

Comment:
**Suggestion**: Use another color for info messages

You're already using Yellow for `Warning`.

Reply:
Changed to green